client/my-sites/stats/activity-log-banner/style.scss

```diff
@@ -71,3 +71,14 @@
 	margin-top: -8px;
 	margin-right: -8px;
 }
+
+.activity-log-banner__progress-bar--queued.is-pulsing .progress-bar__progress {
```

Comment:
It does allow you to modify the background color, but it would be nice have our standard `error` & co. classes/colors on the bar itself.